src/main/resources/swagger/api-docs.yaml

```diff
@@ -7182,6 +7183,26 @@ paths:
             text/plain:
               schema:
                 type: string
+  /register/termsofservice:
```

Comment:
I believe this needs to start with `/register` as I was having trouble with it when it didn't, but open to changing the rest of the path if there's a better one

Reply:
I'm leaning towards wanting to keep the symmetry with the underlying Sam path, `/register/user/v1/termsofservice`